### PR TITLE
Portuguese noun chunks review

### PR DESCRIPTION
**Pull request by DuyguA**
## Description
Noun chunker for Portuguese language

### Types of change

Feature

## Checklist

- [x] I confirm that I have the right to submit this contribution under the project's MIT license.
- [x] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

File: spacy/tests/lang/pt/test_noun_chunks.py [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+        ),
+        # alguns fazendeiros muito ricos -> alguns fazendeiros muito ricos             one determiner + one noun + one adjective qualified by an adverb
+        (
+            ['alguns', 'fazendeiros', 'muito', 'ricos'],
+            [1, 1, 3, 1],
+            ['det', 'ROOT', 'advmod', 'amod'],
+            ['DET', 'NOUN', 'ADV', 'ADJ'],
+            [(0,4)]
+        ),
+        # Eu tenho um cachorro e um gato  -> um cacharo, um gato                                 Two NPs conjuncted
```
Comment (adrianeboyd) [inline spacy/tests/lang/pt/test_noun_chunks.py:113]:
```suggestion
        # Eu tenho um cachorro e um gato  -> Eu, um cacharo, um gato                                 Two NPs conjuncted
```

File: spacy/tests/lang/pt/test_noun_chunks.py [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+        # um irmão meu -> um irmão meu
+        (
+            ["um", "irmão", "meu"],
+            [1, 1, 1],
+            ["det", "ROOT", "det"],
+            ["DET", "NOUN", "DET"],
+            [(0, 3)],
+        ),
+        # two determiners + noun
+        # o meu pai -> o meu pai
```
Comment (damian-romero) [inline spacy/tests/lang/pt/test_noun_chunks.py:46]: Nice!

File: spacy/tests/lang/pt/test_noun_chunks.py [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+        # det + adj + noun
+        # Uma boa ideia -> Uma boa ideia
+        (
+            ['uma', 'boa', 'ideia'],
+            [2, 2, 2],
+            ["det", "amod", "ROOT"],
+            ["DET", "ADJ", "NOUN"],
+            [(0,3)]
+        ),
+        # multiple adjectives
```
Comment (damian-romero) [inline spacy/tests/lang/pt/test_noun_chunks.py:99]: I also tested the `det + adj + noun + adj` case that didn't work so well in Spanish and it worked like a charm:

```
        # Uma boa gatinha parda -> Uma boa gatinha parda
        (
            ['uma', 'boa', 'gatinha', 'parda'],
            [2, 2, 2, 2],
            ["det", "amod", "ROOT", "amod"],
            ["DET", "ADJ", "NOUN", "ADJ"],
            [(0,4)]
        ),
```
No issues :)

**Review by damian-romero — APPROVED** [review]
Looks good :)

I'll keep thinking about examples. Portuguese is not my native language after all :P

**Pull request merged**